game/screen_sing.cc

```diff
@@ -40,6 +40,11 @@ ScreenSing::ScreenSing(std::string const& name, Audio& audio, Database& database
 	m_selectedTrack(TrackName::LEAD_VOCAL)
 {}
 
+bool ScreenSing::singingDuet() {
```

Comment:
this renders this function completely useless, so unless you're going to add the comparison back in in a later stage and only commented it for debugging, I suggest removing the function entirely to get rid of clutter.

Reply:
I am. That's the whole point of [WIP]